github/utils.go

```diff
@@ -30,6 +33,18 @@ func connect(ctx context.Context, d *plugin.QueryData) *github.Client {
 
 	token := os.Getenv("GITHUB_TOKEN")
 	baseURL := os.Getenv("GITHUB_BASE_URL")
+	appId := os.Getenv("GITHUB_APP_ID")
+	installationId := os.Getenv("GITHUB_APP_INSTALLATION_ID")
+	privateKeyPath := os.Getenv("GITHUB_APP_PEM_FILE")
+
+	var githubAppId, githubInstallationId string
```

Comment:
To maintain uniformity in code, can we also add a variable for private_key_path? Like githubPrivateKeyPath?